patches/chrome-browser-importer-external_process_importer_client.h.patch

```diff
@@ -12,16 +12,18 @@ index 9451b0917536e73b42c596527112d3119b2c4cc5..c835dea8c0e0f2c9c8cb1bd450700977
  struct ImportedBookmarkEntry;
  class InProcessImporterBridge;
 +struct BraveStats;
++struct BraveReferral;
```

Comment:
I can likely clean this up the same way @garrettr did for the Brave Rewards import (https://github.com/brave/brave-core/pull/736)